src/DETHRACE/common/structur.c

```diff
@@ -706,7 +706,15 @@ void DoProgram(void) {
             if (gGame_to_load >= 0) {
                 DoLoadGame();
             } else {
-                DoMainMenuScreen(30000u, 0, 0);
+#ifdef DETHRACE_FIX_BUGS
```

Comment:
what about pulling this define up to the top of the file, or maybe in `const.h` where we've put some others? Not sure which one is "better"

Reply:
I pulled `MAIN_MENU_TIMEOUT` to the top of `structur.c`